controller/status_test.go

```diff
@@ -295,6 +295,14 @@ func Test_Status_AggregateStatus(t *testing.T) {
 			ErrorMatcher: IsInvalidUnitStatus,
 			Expected:     "",
 		},
+		{
+			FC:           "inactive",
+			FD:           "launched",
+			SA:           "active",
+			SS:           "running",
+			ErrorMatcher: nil,
+			Expected:     StatusRunning,
+		},
```

Comment:
Is this actually a test for the former broken behaviour? If so I would like to add a comment pointing to this PR and its discussion so one knows about the importance of this special case.